src/main/java/com/waytta/Builds.java

```diff
@@ -70,13 +74,13 @@ public static void addArgumentsToSaltFunction(String myarguments, JSONObject sal
     }
     
 
-    public static JSONArray runBlockingBuild(Run build, JSONArray returnArray, String myservername, 
-    		String token, JSONObject saltFunc, TaskListener listener, int pollTime, int minionTimeout) throws InterruptedException {
+    public static JSONArray runBlockingBuild(Launcher launcher, Run build, JSONArray returnArray, String myservername, 
```

Comment:
It formally breaks the binary compatibility. It is up to the plugin maintainer to decide if it is an issue or not in the case of this plugin